aom/src/main/java/com/nedap/archie/adl14/ADL14NodeIDConverter.java

```diff
@@ -74,51 +56,52 @@ public ADL14NodeIDConverter(MetaModels metaModels, Archetype archetype, Archetyp
 
     }
 
-
     public ADL14ConversionConfiguration getConversionConfiguration() {
         return conversionConfiguration;
     }
 
     public ADL2ConversionLog convert() {
         metaModels.selectModel(archetype);
+
         correctItemsCardinality(archetype.getDefinition());
         List<String> unnecessaryCodes = findUnnecessaryCodes(archetype.getDefinition(),
                 archetype.getTerminology().getTermDefinitions().get(archetype.getOriginalLanguage().getCodeString()));
         convert(archetype.getDefinition());
-        if(previousConversionApplier != null) {
```

Comment:
You can actually start a conversion without having a previous conversion. It should thus be part of the API, or the readme needs an update on how to initialize this as an empty converter. Making the API call more difficult.

So why was this null-check removed?

Reply:
The archetype languege server does not support storing previous conversion, so does never set a previous conversion applier, for example. Please do not remove.